src/ResourceManager/ContainerInstance/Commands.ContainerInstance/packages.config

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.ContainerInstance" version="1.0.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
```

Comment:
@yolo3301 same comment about removing this reference

Reply:
Removed